src/libs/ReportUtils.js

```diff
@@ -1159,6 +1159,11 @@ function buildOptimisticIOUReportAction(type, amount, currency, comment, partici
     // IOUs of type split only exist in group DMs and those don't have an iouReport so we need to delete the IOUReportID key
     if (type === CONST.IOU.REPORT_ACTION_TYPE.SPLIT) {
         delete originalMessage.IOUReportID;
+        const participantLogins = [currentUserEmail];
+        _.each(participants, (participant) => {
+            participantLogins.push(participant.login);
+        });
+        originalMessage.participants = participantLogins;
```

Comment:
```suggestion
        originalMessage.participants = [currentUserEmail, ..._.pluck(participants, 'login')];
```
NAB: Just a simplification.